### PR TITLE
[Tests] Remove surplus functional tests

### PR DESCRIPTION
**Pull request by aciidb0mb3r**
These tests now have coverage in ConventionTests, functional tests are
not needed because they'll error out before every getting to Build.

### CODE REVIEW

File: Tests/Functional/ValidLayoutTests.swift [hunk 1]
```diff
@@ -34,18 +34,6 @@ class ValidLayoutsTestCase: XCTestCase {
         }
     }
 
-    func testSingleModuleCustomizedName() {
-
-        // Package.swift for a single module with a customized name
-        // names that target after the package name
-
-        runLayoutFixture(name: "SingleModule/CustomizedName") { prefix in
```
Comment (aciidb0mb3r) [inline Tests/Functional/ValidLayoutTests.swift]: This test isn't needed since we don't support empty `Package.swift` anyway

**Comment by ddunbar** [comment]
BTW, did you vet this against code coverage statistics? I'd be a good sanity check that we aren't losing test coverage.

**Comment by aciidb0mb3r** [comment]
Will it be able to detect against functional tests?

**Comment by aciidb0mb3r** [comment]
Doesn't look like we lost coverage:
Before:

After:

**Comment by ddunbar** [comment]
Yeah it should (functional tests)... great thanks for checking!

**Comment by ddunbar** [comment]
LGTM

**Pull request merged**